README.md

```diff
@@ -14,148 +14,80 @@ Metro Board Reports is a member of the [Councilmatic family](https://www.council
 
 ## Setup
 
-**Install OS level dependencies:**
+These days, we run apps in containers for local development. Prefer to run the app locally? See the [legacy setup instructions](https://github.com/datamade/la-metro-councilmatic/blob/b8bc14f6d90f1b05e24b5076b1bfcd5e0d37527a/README.md).
 
-* Python 3.4
-* PostgreSQL 9.4 +
+### Install OS level dependencies:
 
-**Install app requirements**
+* [Docker](https://www.docker.com/get-started)
 
-We recommend using [virtualenv](http://virtualenv.readthedocs.org/en/latest/virtualenv.html) and [virtualenvwrapper](http://virtualenvwrapper.readthedocs.org/en/latest/install.html) for working in a virtualized development environment. [Read how to set up virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/).
-
-Once you have virtualenvwrapper set up, run the following in your terminal:
-
-```bash
-mkvirtualenv la-metro
-git clone git@github.com:datamade/la-metro-councilmatic.git
-cd la-metro-councilmatic
-pip install -r requirements.txt
-```
-
-Afterwards, whenever you want to use this virtual environment, run `workon la-metro`.
-
-**Special installation requirements**
-
-Metro runs a custom script that converts board-report attachments into plaintext. The script uses `textract` for conversions. Mac users can [follow the instructions given by `textract`](http://textract.readthedocs.io/en/stable/installation.html#osx).
-
-For Ubuntu users and on the server, the `textract` installation instructions require a little remediation. Follow these steps:
-
-(1) [Install all the dependencies](http://textract.readthedocs.io/en/stable/installation.html#ubuntu-debian).
-
-(2) `textract` fails when installing `pocketsphinx`. Happily, `textract` does not need this dependency for converting PDFs or word documents into plain text. [This issue offers a script that installs a "fake pocketsphinx"](https://github.com/deanmalmgren/textract/pull/178). Do as the issue instructs.
-
-(3) You're ready! Inside your virtualenv execute: `pip install textract==1.6.1`.
-
-
-**Create your settings file**
+### Run the application
 
 ```bash
-cp councilmatic/settings_deployment.py.example councilmatic/settings_deployment.py
+docker-compose up -d
```

Comment:
Maybe this is asking too much, but I was a bit confused about whether the app was running because I was used to seeing the solr logs outputted onto my terminal. Maybe a short note about what a successful `up` would result in could help clarify this? Also, instructions for stopping the container would be useful too.